#point in image coordinates
geometry_msgs/Point image_point
---
#point in world coordinates
geometry_msgs/Point world_point
